wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -2057,11 +2156,23 @@ func OpenAICommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (sstor
 	if err != nil {
 		return nil, fmt.Errorf("openai error, cannot make dyn cmd")
 	}
+	prompt := []packet.OpenAIPromptMessageType{{Role: sstore.OpenAIRoleUser, Content: promptStr}}
+	if resolveBool(pk.Kwargs["cmdinfo"], false) {
```

Comment:
I think eventually we also send the current command as a kwarg, then we can do the prompt engineering

Reply:
Addressed